2009-05-02  Ralf Wildenhues  <Ralf.Wildenhues@gmx.de>

	Skip localization test when setlocale is not functional.
	* tests/localization.at (localized compiler messages): Skip
	test when there is a warning about `LC_ALL'.
	* THANKS: Update.
	Report by Maciej Helminiak.

Index: libtool-2.2.6b/tests/localization.at
===================================================================
--- libtool-2.2.6b.orig/tests/localization.at	2009-11-16 13:11:59.000000000 +0000
+++ libtool-2.2.6b/tests/localization.at	2009-12-09 18:49:08.000000000 +0000
@@ -47,6 +47,9 @@
 mv -f stderr expected-stderr
 AT_CHECK([$LIBTOOL --mode=compile --tag=CC $CC $CPPFLAGS $CFLAGS -c a.c || exit 1],
 	 [1], [stdout], [stderr])
+# Avoid this error:
+#   /bin/sh: warning: setlocale: LC_ALL: cannot change locale (de_DE)
+AT_CHECK([grep LC_ALL stdout stderr && exit 77], [1])
 AT_CHECK([diff expected-stderr stderr])
 LTBASE=`$ECHO "$LIBTOOL" | sed 's,^.*/,,'`
 AT_CHECK([grep -v "^$LTBASE: compile" stdout | diff expected-stdout -])
